pkgs/applications/emulators/duckstation/default.nix

```
@@ -20,18 +20,17 @@
 , vulkan-loader
 , wayland
 , wrapQtAppsHook
-, enableWayland ? true
 }:
 
-stdenv.mkDerivation (finalAttrs: {
+llvmPackages_16.stdenv.mkDerivation (finalAttrs: {
   pname = "duckstation";
-  version = "unstable-2023-09-30";
+  version = "unstable-2024-01-08";
```

Please follow the "new" unstable version guidelines: https://github.com/NixOS/nixpkgs/blob/640014bd83bd2e50bf0ab6c3460fe0d50bba9f9d/pkgs/README.md#package-naming

```suggestion
  version = "0.1-6232-unstable-2024-01-08";
```

---

They aren't tagging it afaicr - they merely use it as a counter (generated by Git, but a counter nonetheless).

Further, such counter can't be scripted easily: it would require to download the `.git` directory.

---

> it would require to download the `.git` directory.

That was also a problem before.
https://github.com/NixOS/nixpkgs/blob/2adf53bea0788813ffe187d38fdd5544425427fb/pkgs/applications/emulators/duckstation/default.nix#L40-L46

---

I was talking more about the updating scripts.